files_deleted.txt

```diff
@@ -0,0 +1,24 @@
+/Users/kimlew/Sites/bash_projects/test_run/IMG_0375 (1).jpg
```

Comment:
Did you want to commit this file to your repository?

If not, I can show you how to remove it from your PR, though it takes a couple of steps.

Reply:
@kcharter No, that isn't supposed to be part of the commit. Yes, I should remove it.